packages/core/contracts/data-verification-mechanism/implementation/Staker.sol

```diff
@@ -301,32 +301,39 @@ abstract contract Staker is StakerInterface, Ownable, Lockable, MultiCaller {
     /**
      * @notice  Determine the number of outstanding token rewards that can be withdrawn by a voter.
      * @param voter the address of the voter.
-     * @return uint256 the outstanding rewards.
+     * @return uint128 the outstanding rewards.
      */
-    function outstandingRewards(address voter) public view returns (uint256) {
+    function outstandingRewards(address voter) public view returns (uint128) {
         VoterStake storage voterStake = voterStakes[voter];
 
         return
-            ((voterStake.stake * (rewardPerToken() - voterStake.rewardsPaidPerToken)) / 1e18) +
-            voterStake.outstandingRewards;
+            uint128(
+                ((uint256(voterStake.stake) * (rewardPerToken() - voterStake.rewardsPaidPerToken)) / 1e18) +
+                    voterStake.outstandingRewards
+            );
```

Comment:
An alternative to avoid increase in lines would be to leave `uint256` both for `outstandingRewards` and `rewardPerToken` methods and downcast their return values elsewhere. Just did go that route for the sake of consistency.